providers/src/airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -125,7 +125,7 @@
     "example_display_video_sdf",
     start_date=START_DATE,
     catchup=False,
-) as dag3:
+) as dag_example_display_video_sdf:
```

Comment:
Nit: unrelated changes in this file

Reply:
it's so that they can be imported in a test 

see /Users/dstandish/code/airflow/providers/tests/google/marketing_platform/operators/test_display_video_system.py